### DISCUSSION TITLE
Unauthorized access error

### DISCUSSION THREAD

**Discussion by Alferedon** [neos-sdi/adfsmfa#260] [Q&A] [answered]
Hi
in a fresh installation of ADFSMFA, after configuring the mfa, getting unauthorized access error. I have tested all the template but seems like the problem that i have faced is somewhere else. 
![1](https://user-images.githubusercontent.com/41039754/195980267-4cf7b60e-c9e9-4d19-b39e-e830e1767443.PNG)
![2](https://user-images.githubusercontent.com/41039754/195980270-26f593b4-cbfd-4a10-9afb-c4990a9691c1.PNG)
![3](https://user-images.githubusercontent.com/41039754/195980271-c0847404-03dc-4bfc-97ab-067949d9e14b.PNG)

**Comment by redhook62** [comment, marked as answer]
Hi,

Your User has not registered for MFA, and the policy you have activated should require administrative registration rather than letting users self-register.

see : [Policies](https://github.com/neos-sdi/adfsmfa/wiki/05-General%20Settings#user-features-policies-templates)

regards
